app/models/service.rb

```diff
@@ -0,0 +1,4 @@
+class Service < ApplicationRecord
+  has_one :user
+  validates :category, uniqueness: true, presence: true
```

Comment:
CATEGORIES = %w(construction education health programing)
  validates :category, presence: true, inclusion: { in: CATEGORIES,
    message: "%{value} is not permited category" }